src/components/molecules/SpaceEditForm/SpaceEditForm.scss

```diff
@@ -1,16 +1,30 @@
 @import "scss/constants.scss";
 
-.EditRoomForm {
+$spaces-footer-height: 75px;
```

Comment:
Shouldn't this already be covered by or at least moved to this block https://github.com/sparkletown/sparkle/blob/staging/src/scss/constants.scss#L69-L82 ?

Reply:
As a reference (might as well copy from): https://github.com/sparkletown/sparkle/blob/staging/src/components/organisms/AdminVenueView/components/TabFooter/TabFooter.scss#L14-L19

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2312/commits/0023b3b4d849d0af4ae51d5af3ca558495cb927e